sql/src/main/java/io/crate/metadata/cluster/DDLClusterStateHelpers.java

```diff
@@ -90,4 +92,21 @@ static IndexTemplateMetaData templateMetaData(MetaData metaData, RelationName re
         String templateName = PartitionName.templateName(relationName.schema(), relationName.name());
         return metaData.templates().get(templateName);
     }
+
+    static void renameIndices(MetaData metaData,
+                              MetaData.Builder mdBuilder,
+                              ClusterBlocks.Builder blocksBuilder,
+                              Index[] sourceIndices,
+                              String[] targetIndices) {
+        for (int i = 0; i < sourceIndices.length; i++) {
+            Index index = sourceIndices[i];
```

Comment:
minor: I'd rename `index` to `source` or something similar as it's a bit hard to follow - especially as below at some point you see `mdBuilder.remove(index.getName());` and think .. which index is that? :)